portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -374,6 +374,38 @@ definition {
 		AssertElementPresent(locator1 = "CPSAMLAdminIdentityProviderConnection#IDENTITY_PROVIDER_CONNECTION_ERROR_NO_MAPPING_SELECTED");
 	}
 
+	@description = "LPS-123218 TC-2 Assertion for: first name cannot be used for user matching even if LDAP import is enabled."
+	@priority = "4"
+	test AssertFirstNameCannotBeUsedForUserMatchingEvenWithLDAPEnabled {
+		property apacheds.enabled = "true";
+		property app.server.bundles.size = "0";
+		property portal.upstream = "feri";
```

Comment:
Please remove the custom property or revert it to true

Reply:
My bad! Removed already.